python/dgl/graphbolt/minibatch.py

```diff
@@ -473,6 +474,35 @@ def apply_to(x, device):
 
         return self
 
+    def to_pyg_adapter(self, device):
+        def construct_edge_index(sampled_subgraph, device):
```

Comment:
move the import to this method to avoid import when unnecessary.

Reply:
sorry, what does this mean